### PR TITLE
Datepicker input

### PR DESCRIPTION
**Pull request by MartinKokesCz**
Based on issue #946 I have prepared the simple HTML input type="date" for date and date range filter.

You can try it with my fork of datagrid-skeleton

https://github.com/MartinKokesCz/datagrid-skeleton/tree/next

### CODE REVIEW

**Comment by radimvaculik** [comment]
@MartinKokesCz Please check Phpstan issues related to changes. I will fix the rest of codesniffer/phpstan issues.

**Comment by radimvaculik** [comment]
@MartinKokesCz Please, rebase, I fixed all related issues in `next` branch.

**Comment by MartinKokesCz** [comment]
@radimvaculik maybe it is ready now ?

File: src/Column/ColumnDateTime.php [hunk 1]
```diff
@@ -47,4 +47,9 @@ public function setFormat(string $format): self
 		return $this;
 	}
 
+	public function getFormat(): string
```
Comment (radimvaculik) [inline src/Column/ColumnDateTime.php:50]: No need anymore. @MartinKokesCz

**Review by radimvaculik — CHANGES_REQUESTED** [review]

File: src/Column/ColumnDateTime.php (hunk 1 above)
Comment (MartinKokesCz) [reply src/Column/ColumnDateTime.php]: @radimvaculik ok, removed. Thanks

**Pull request closed without merging**